src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,116 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  font-family: Roboto, sans-serif;
+  border: 1px solid $main-border-color;
+  background-color: $main-background-color;
+  border-radius: 5px;
+  color: $main-text-color;
+  transition: transform 0.3s ease-in-out;
+
+  &__img-link {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-top: 0;
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 0;
+    margin-bottom: 16px;
```

Comment:
use `margin: 0 0 16px` option or reset margins for all p in `reset.scss` file.
in all such cases